src/vector/mod.rs

```diff
@@ -182,7 +182,7 @@ impl<A: Clone> Vector<A> {
     ///
     /// Note that if you didn't specifically construct it with a pool, you'll
     /// get back a reference to a pool of size 0.
-    #[cfg_attr(not(feature = "pool"), doc = "hidden")]
+    #[cfg_attr(not(feature = "pool"), doc("hidden"))]
```

Comment:
You want `doc(hidden)` I think.
```suggestion
    #[cfg_attr(not(feature = "pool"), doc(hidden))]
```
https://doc.rust-lang.org/rustdoc/the-doc-attribute.html?highlight=hidden#dochidden